standards/cryptography.md

```diff
@@ -0,0 +1,57 @@
+---
+expires: 2019-03-15
+---
+# Cryptography
+
+## The base principle
+
+All data **must** employ adequate and proportionate cryptography to preserve confidentiality and integrity whether data is at-rest or in-transit.
+
+## In-transit
+
+In-transit encryption techniques can both protect data during transit through cryptography but also help facilitate the establishing of identity of devices on one or more sides of the connection.
+
+### Transport Layer Security (TLS)
+
+The [National Cyber Security Centre (NCSC)](https://www.ncsc.gov.uk/) have published information on good TLS configurations [https://www.ncsc.gov.uk/guidance/tls-external-facing-services](https://www.ncsc.gov.uk/guidance/tls-external-facing-services).
+
+In general, subject to document exceptions (such as end-user needs and required legacy backwards compatiability)
+
+#### Testing
+
+Tools such as [Qualys SSL Server Test](https://www.ssllabs.com/ssltest/) and Check TLS services from [checktls.com](http://www.checktls.com/index.html) **should** be used where applicable to help identify most common issues and configuration problems.
+
+While these tools are not a replacement for skilled testing, the outputs of these tools can help you identify inefficient or insecure configurations which should be considered for remediation.
+
+### Internet protocol security (IPsec)
+
+NCSC have published information on good IPsec configurations [https://www.ncsc.gov.uk/guidance/using-ipsec-protect-data](https://www.ncsc.gov.uk/guidance/using-ipsec-protect-data).
+
+## At-rest
+
+At-rest encryption techniques can protect data while being stored and even during some processing. At-rest techniques usually protect against physical theft or attack methods.
+
+### Portable storage
+
+Portable storage such as CDs, DVDs and USB sticks can be safely used to move data. As usual, data must be adequately protected based on the overall governance and information risk requirements.
```

Comment:
I think there's something needed here about it is our preference that we prefer online transfers compared to the use of physical media (even if encrypted). Also might need something in here about key management / exchanging passwords etc for media.

Reply:
> I think there's something needed here about it is our preference that we prefer online transfers compared to the use of physical media (even if encrypted).

PR inbound

> Also might need something in here about key management / exchanging passwords etc for media.

I'm out of words.